src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js

```diff
@@ -34,6 +34,8 @@ const Datestamp = ({ date }: { date: ISODate | Date }) => {
     return <ReadableTime timestamp={new Date(date).getTime()} alwaysShowTime relativeThreshold={0} />;
 };
 
+// TODO: Move to a separate component with its own Props type
+/* eslint-disable react/prop-types */
 const AvatarDetails = React.memo(({ user, status, completedAt, className }) => {
```

Comment:
```suggestion
type DetailsProps = {
    user: { name: string },
    status: $Keys<statusMessages>,
    completedAt?: ?Date,
    className?: string,
};
const AvatarDetails = React.memo(({ user, status, completedAt, className }: DetailsProps) =>
```

Reply:
The above is going to throw errors due to out-of-order prop keys, I think. Can we handle this cleanup in a follow-up?

Reply:
The above is going to throw errors due to out-of-order prop keys, I think. Can we handle this cleanup in a follow-up?

Reply:
Yes, if you want.